extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss

```diff
@@ -162,19 +162,57 @@
 		@include column-count(4);
 		display: none;
 
-		label {
-			display: inline-block;
-			position: relative;
-			width: 100%;
+		span {
+			display: block;
+			margin-left: 32px;
 		}
 
-		input {
-			position: absolute;
-		}
+		.checkbox-styled {
+			float: left;
+			position: relative;
+			width: 18px;
+
+			label {
+				border: 1px solid $wds-color-light-gray;
+				border-radius: 1px;
+				cursor: pointer;
+				height: 18px;
+				left: 0;
+				position: absolute;
+				top: 0;
+				width: 18px;
+
+				&:after {
```

Comment:
Double colon (`&::after {`) is preferred for pseudo-elements. https://developer.mozilla.org/en/docs/Web/CSS/Pseudo-elements#Notes